packages/puppeteer-core/src/cdp/Browser.ts

```diff
@@ -347,10 +347,19 @@ export class CdpBrowser extends BrowserBase {
     return await this.#defaultContext.newPage();
   }
 
-  async _createPageInContext(contextId?: string): Promise<Page> {
+  async _createPageInContext(
+    contextId?: string,
+    options?: CreatePageOptions,
+  ): Promise<Page> {
+    const hasTargets =
+      this.targets().filter(t => {
```

Comment:
Out of curiosity I understand that needs to check if there are already open pages,
is it possible there can be other targets and then this creation will fail?

Reply:
I think if there is any target for the context, the browser profile object in chromium would be instantiated and newWindow argument would work. It only does not work if there is nothing in the browser context at all.

Reply:
Thanks for the info!